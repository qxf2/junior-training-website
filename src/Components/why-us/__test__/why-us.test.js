import React from 'react';
import ReactDOM from 'react-dom';
import WhyUs from '../why-us';

import { render, cleanup } from '@testing-library/react';
import "@testing-library/jest-dom/extend-expect";

import renderer from "react-test-renderer";

afterEach(cleanup);

it('renders WhyUs crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<WhyUs/>, div);
});

it("renders Benefits correctly", () => {
   const { getByTestId }  = render(<WhyUs/>)
   expect(getByTestId('why-us'));
});

it("Why-us.js matches snapshot", () =>{
  const tree = renderer.create(<WhyUs/>).toJSON();
  expect(tree).toMatchSnapshot();
});