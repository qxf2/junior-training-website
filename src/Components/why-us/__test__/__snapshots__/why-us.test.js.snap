// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Why-us.js matches snapshot 1`] = `
<section
  data-testid="why-us"
  id="why-us"
>
  <div
    className="react-reveal row"
    style={
      Object {
        "opacity": undefined,
      }
    }
  >
    <h4
      className="react-reveal"
      style={
        Object {
          "opacity": undefined,
        }
      }
    >
      Why us?
    </h4>
    <div
      className="twelve columns collapsed"
    >
      <div
        className="bgrid-quarters s-bgrid-thirds cf"
      >
        <ul>
          <li>
             Professional testers teach you 
          </li>
          <li>
             No spoonfeeding 
          </li>
          <li>
             Practical ... not theoretical 
          </li>
          <li>
             No school like approach 
          </li>
          <li>
             No syllabus or book knowledge 
          </li>
          <li>
             Real application 
          </li>
        </ul>
      </div>
    </div>
  </div>
</section>
`;
